src/css/styles/_top.scss

```diff
@@ -41,11 +46,34 @@
   padding: 1rem;
 }
 
+.l-center-vertical {
+  position: relative;
+  top: 50%;
+  transform: translateY(-50%);
+}
+
+.l-center-vertical--banner {
+  padding-top: 52px;
+}
```

Comment:
Correct me if I am wrong: 52px is height of menu, isn't it?

Reply:
Unfortunately, scss vars cannot be used inside `calc(*)` expression, on new branch I will get rid of this variable and return to using pixels in its places of use.